api/src/main/kotlin/edu/wgu/osmt/api/model/ApiCollectionV2.kt

```diff
@@ -0,0 +1,83 @@
+package edu.wgu.osmt.api.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import edu.wgu.osmt.collection.Collection
+import edu.wgu.osmt.collection.CollectionDao
+import edu.wgu.osmt.config.AppConfig
+import edu.wgu.osmt.db.PublishStatus
+import edu.wgu.osmt.richskill.RichSkillDescriptor
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+
+@JsonInclude(JsonInclude.Include.ALWAYS)
+class ApiCollectionV2(
+    private val collection: Collection,
+    private val ss: List<RichSkillDescriptor>,
+    private val appConfig: AppConfig
+) {
+    @get:JsonProperty
+    val id: String
+        get() = collection.canonicalUrl(appConfig.baseUrl)
+
+    @JsonProperty("@context")
+    val context = appConfig.rsdContextUrl
+
+    @JsonProperty
+    val `type` = "RichSkillCollection"
```

Comment:
IDE says that ` is redundant, is this correct?

Reply:
this was removed from this implementation, so closing this comment.